html/semantics/interactive-elements/the-dialog-element/centering-iframe.sub.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<meta charset=utf-8>
+<title>dialog element centered frame</title>
+<style>
+  html {
+    writing-mode: {{GET[html-writing-mode]}}
+  }
+
+  dialog, body {
```

Comment:
We can't use `body` here because 'writing-mode' propagates from `body` to the viewport. Should use a `div`. Also don't need to put `dialog` in the selector since it's inherited.